File: dsa/sorting/SortHelper.java
package sorting;

import java.util.Scanner;

// Function to take create and take input in an array
public class SortHelper {
    static Scanner sc = new Scanner(System.in);

    public static int[] takeInput() {
        System.out.print("Enter the length of the Array -> ");
        int len = sc.nextInt();
        int[] arr = new int[len];
        arr = new int[len];
        System.out.print("Enter the elements -> ");
        for (int i = 0; i < len; i++)
            arr[i] = sc.nextInt();
        return arr;
    }

    // Fn to take input for sort options
    public static int sort() {
        System.out.print("1 for Ascending\n" +
                "2 for Descending\n" +
                "Choose -> ");
        int sort = sc.nextInt();
        return sort;
    }

    // Fn to print the Sorted Array
    public static void printSortedArray(int[] arr) {
        System.out.print("Array after sorting -> [");
        for (int i : arr)
            System.out.print(i + ", ");
        System.out.println("\b\b]");
    }
}

File: DsaWithCpp/Sorting/SortHelper.cpp
#include <iostream>
#include <vector>
using namespace std;

static void takeInput(vector<int> &arr)
{
    int len;
    cout << "Enter the length of the Array -> ";
    cin >> len;
    cout << "Enter the Elements -> ";
    for (int i = 0; i < len; i++)
    {
        int temp;
        cin >> temp;
        arr.push_back(temp);
    }
}

static int sort()
{
    cout << "1 for Ascending\n"
         << "2 for Descending\n"
         << "Choose -> ";
    int sort;
    cin >> sort;
    return sort;
}

static void printSortedArray(vector<int> arr)
{
    cout << "Array after sorting -> [ ";
    for (int i = 0; i < arr.size(); i++)
        cout << arr[i] << ", ";
    cout << "\b\b ]" << endl;
}
